core/common/src/main/java/alluxio/util/network/NettyUtils.java

```diff
@@ -149,9 +150,16 @@ public static void disableAutoRead(Channel channel) {
    * @return true if the domain socket is enabled on this client
    */
   public static boolean isDomainSocketSupported(WorkerNetAddress workerNetAddress) {
-    return workerNetAddress.getHost().equals(NetworkAddressUtils.getClientHostName())
-        && !workerNetAddress.getDomainSocketPath().isEmpty()
-        && CHANNEL_TYPE == ChannelType.EPOLL;
+    boolean isDomainSocketSupported = !workerNetAddress.getDomainSocketPath().isEmpty();
```

Comment:
we could simplify this by immediately returning `false` if `workerNetAddress.getDomainSocketPath().isEmpty()`